pages/js-templates/template.js.hbs

```diff
@@ -0,0 +1,16 @@
+/**
```

Comment:
Why ? we have separate templates we can make use of existing components to be created at /template/*

Reply:
Done. Referring to existing component templates.